js/assessment.js

```diff
@@ -242,7 +242,7 @@ define([
 
         get: function(id) {
             if (id === undefined) {
-                return this._assessments.slice(0);
+                return this._assessments[0];
```

Comment:
Good catch @oliverfoster.  I didn't actually need a `getFirst()`.  The change in 4ec4121 seems to do enough.